experimental/rockbuilder/rockbuilder.py

```diff
@@ -196,100 +248,94 @@ def verify_build_env(args):
             sys.exit(1)
 
     # check rocm
-    if "ROCM_HOME" not in os.environ:
-        rocm_home_root_path = Path(rock_builder_home_dir) / "../../build/dist/rocm"
-        rocm_home_root_path = rocm_home_root_path.resolve()
-        # print("ROCM_HOME: " + rocm_home_root_path)
-        if rocm_home_root_path.exists():
+    if "ROCM_HOME" in os.environ:
+        rocm_home_root_path = Path(os.environ["ROCM_HOME"])
+    else:
+        rocm_home_root_path = rock_builder_home_dir / "../../build/dist/rocm"
```

Comment:
When making decisions based on inputs or fallback locations, those decisions should be logged ("I didn't see this variable set, so I'm going to try this default location... oh, the default location does not exist, that's an error")